perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py

```diff
@@ -7,8 +7,13 @@
 from typing import Any, Dict, List, Tuple
 
 import numpy as np
-from aria_data_utils.detector_wrappers.generic_detector_interface import GenericDetector
-from aria_data_utils.image_utils import centered_heuristic, check_bbox_intersection
+from perception_and_utils.perception.detector_wrappers.generic_detector_interface import (
+    GenericDetector,
+)
+from perception_and_utils.utils.image_utils import (
+    centered_heuristic,
```

Comment:
I wonder if we should change this name to `centered_object_detection_heuristic` to make the usage more explicit?

Reply:
updated.